bitmap.go

```diff
@@ -53,15 +53,15 @@ func (r *Renderer) drawBitmap(g shaping.Glyph, bitmap api.GlyphBitmap, img draw.
 			}
 		}
 
-		rect := image.Rect(int(x), int(y), int(x+fixed266ToFloat(g.Width)*r.PixScale), int(y-fixed266ToFloat(g.Height)*r.PixScale))
+		rect := image.Rect(int(x), int(y-fixed266ToFloat(g.YBearing)*r.PixScale), int(x+fixed266ToFloat(g.Width)*r.PixScale), int(y))
 		scale.NearestNeighbor.Scale(img, rect, sub, sub.Bounds(), draw.Over, nil)
 	case api.JPG, api.PNG, api.TIFF:
 		pix, _, err := image.Decode(bytes.NewReader(bitmap.Data))
 		if err != nil {
 			return err
 		}
 
-		rect := image.Rect(int(x), int(y), int(x+fixed266ToFloat(g.Width)*r.PixScale), int(y-fixed266ToFloat(g.Height)*r.PixScale))
+		rect := image.Rect(int(x), int(y-fixed266ToFloat(g.YBearing)*r.PixScale), int(x+fixed266ToFloat(g.Width)*r.PixScale), int(y))
```

Comment:
The aspect ratio looks wrong - should `g.Width` be `g.XBearing` so we are using the same coordinate on both directions?

Reply:
Oops, it isn't that simple sorry. But I think it needs to know the ratio of height to YBearing and then apply that to the X axis